src/test/java/org/sonarsource/sonarlint/ls/ServerMainTests.java

```diff
@@ -20,121 +20,76 @@
-        "[-analyzers path/to/analyzer1.jar [path/to/analyzer2.jar] ...] " +
-        "[-extraAnalyzers path/to/analyzer3.jar [path/to/analyzer4.jar] ...]" + System.lineSeparator());
-  }
-
-  @Test
-  void testInvalidPortArgument() {
-
-    var thrown = assertThrows(RuntimeException.class, () -> {
-      underTest.startLanguageServer("not_a_number");
-    });
-
-    assertThat(thrown).hasMessage("exit called");
-    assertThat(err.toString(StandardCharsets.UTF_8))
-      .contains("Invalid port provided as first parameter");
-  }
-
-  @Test
-  void testInvalidPluginPath() {
-
-    var thrown = assertThrows(RuntimeException.class, () -> {
-      underTest.startLanguageServer("1", "-analyzers", INVALID_PATH);
-    });
-
-    assertThat(thrown).hasMessage("exit called");
-    assertThat(err.toString(StandardCharsets.UTF_8))
-      .contains("Invalid argument at position 3. Expected a path.");
-  }
-
-  @Test
-  void testInvalidExtraPluginPath() {
-
-    var thrown = assertThrows(RuntimeException.class, () -> {
-      underTest.startLanguageServer("1", "-analyzers", "folder/analyzer1.jar", "-extraAnalyzers", INVALID_PATH);
-    });
-
-    assertThat(thrown).hasMessage("exit called");
-    assertThat(err.toString(StandardCharsets.UTF_8))
-      .contains("Invalid argument at position 5. Expected a path.");
-  }
-
-  @Test
-  void testExtractingAnalyzersPositive() {
-    var args = new String[] {"-analyzers", "folder/analyzer1.jar", "folder/analyzer2.jar"};
-    var paths = underTest.extractAnalyzers(args);
-    assertThat(paths).containsExactly(Paths.get("folder/analyzer1.jar"), Paths.get("folder/analyzer2.jar"));
-  }
-
-  @Test
-  void testExtractingAnalyzersExitsIfNoKey() {
-    var args = new String[] {"folder/analyzer1.jar", "folder/analyzer2.jar"};
-
-    assertThrows(RuntimeException.class, () -> underTest.extractAnalyzers(args));
-  }
-
-  @Test
-  void testExtractingAnalyzersExitsOnMalformedPath() {
-    var args = new String[] {"-analyzers", INVALID_PATH};
-
-    assertThrows(RuntimeException.class, () -> underTest.extractAnalyzers(args));
-  }
-
-  @Test
-  void testExtractingExtraAnalyzersPositive() {
-    var args = new String[] {"-extraAnalyzers", "folder/analyzer1.jar", "folder/analyzer2.jar"};
-    var paths = underTest.extractExtraAnalyzers(args);
-    assertThat(paths).containsExactly(Paths.get("folder/analyzer1.jar"), Paths.get("folder/analyzer2.jar"));
-  }
-
-  @Test
-  void testExtractingExtraAnalyzersReturnsEmptyListIfNoKey() {
-    var args = new String[] {"folder/analyzer1.jar", "folder/analyzer2.jar"};
-
-    var paths = underTest.extractExtraAnalyzers(args);
-    assertThat(paths).isEmpty();
-  }
-
-  @Test
-  void testExtractingExtraAnalyzersExitsOnMalformedPaths() {
-    var args = new String[] {"-extraAnalyzers", INVALID_PATH};
-
-    assertThrows(RuntimeException.class, () -> underTest.extractExtraAnalyzers(args));
-  }
+   private static final String INVALID_PATH = "\0invalid?;path\n";
```

Comment:
Formatting issue. SonarSource coding style is to use 2 spaces for indentation

Reply:
I applied the Eclipse setting in [eclipes.jdt.ls](https://github.com/eclipse/eclipse.jdt.ls) and formatted all touched files. But it rearranged some static imports. Is it intended to have the static imports at the top?

Reply:
Are you using https://github.com/SonarSource/sonar-developer-toolset/blob/master/eclipse/sonar.importorder? Static imports should all be after other imports.